package/README.md

```diff
@@ -0,0 +1,39 @@
+# tlg.catalog.pkg
+
+This is a dummy package used for testing purposes. For the actual content please navigate to the [book directory](../book).
+
+The package itself is relatively empty and most of the logic lives in the `tests/` subdirectory. The main goal is to reuse state of art GitHub Actions and other CI/CD tools designed for package development. This includes `lintr`, `styler`, `spelling`, and `testthat2` snapshot testing.
+
+## Regression tests
+
+### Overview
+
+The book component is symlinked into `tests/testthat/_book` subdirectory (no need to copy&paste). As a part of test setup, the whole book will be regenerated. A dedicated `knitr` hooks will be invoked to save relevant output objects as `.Rds` files. During actual testing, these files are loaded and compared with the expected ones using `testthat2` snapshot testing.
+
+A _standard_ render will be unaffected. These hooks will be used only during testing.
+
+### How to use?
+
+* At the beggining of the article, add the following: `{{< include ../../test-utils/envir_hook.qmd >}}`. It will register the `knitr` hooks for further use.
+* For code chunks without output - do nothing.
+* For code chunks with output:
+  * Inside chunk, make sure you save the output into the object (e.g. `table <- foo(...); table` instead of `foo(...)`). Don't forget to print the object at the end of the chunk to make it visible in the article.
+  * In the chunk header, use a dedicated `test` chunk template giving previously described object name a new, unique(!) name using following syntax: `test = list(<new name> = "<old name>")` (e.g. use `table` object from the current chunk and name it as `result_v1`: `{r, test = list(result_v1 = "table")}`).
+* At the end of the article (after all chunks with outputs), add `{{< include ../../test-utils/save_results.qmd >}}` to save all objects as `.Rds` files.
+
+This logic had been implemented in many articles already. Please refer to the existing ones for examples.
```

Comment:
```suggestion
This logic has been implemented in many articles already. Please refer to the existing ones for examples.
```

"Many"? Meaning some but not all?

Reply:
Presumably all but there might be some exceptions. It's nowhere enforced.